hello.py

```diff
@@ -1,17 +1,26 @@
-import time
 import gradio as gr
-import numpy as np
 import json
 from prototype import chat
-from constants import states
+from uszipcode import SearchEngine
```

Comment:
Just from a quick look at the code, the one thing you may have forgotten is to modify the requirements.txt to include the uszipcode package. I'll test.